PassleSync.Core/ContentFinders/PassleAuthorContentFinder.cs

```diff
@@ -9,8 +9,9 @@ public PassleAuthorContentFinder(ConfigService configService) : base(configServi
         {
         }
 
-        protected override string UrlPrefix => _configService.AuthorPermalinkPrefix;
-        protected override string ShortcodeName => "Shortcode";
+        protected override string UrlTemplate => _configService.PersonPermalinkTemplate;
+        protected override string ShortcodeName => "PersonShortcode";
```

Comment:
I think this will break things
This property is used to refer to the `Shortcode` property of the author data returned by the API
We may need an additional property for the name of the person shortcode template variable
I'd refer to how the WP plugin handles this